support/android/apk/servoapp/src/main/java/org/mozilla/servo/MainActivity.java

```diff
@@ -162,6 +163,19 @@ public void onHistoryChanged(boolean canGoBack, boolean canGoForward) {
         mCanGoBack = canGoBack;
     }
 
+    @Override
+    public boolean onAllowNavigation(String url) {
+        if (url.startsWith("market://")) {
+            try {
+                startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse(url)));
+                return false;
+            } catch (Exception e) {
+                Log.e("onAllowNavigation", e.toString());
```

Comment:
@paulrouget 
Do we need to do something here? 

To my knowledge, this exception will be thrown on 2 cases. If there is no app to handle the URL (play store is not installed) or there is a problem in parsing URL.

Reply:
Printing a warning and letting it return true sounds about right.

Reply:
I'll update this with `Log.w`. 👍